android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -206,11 +208,9 @@ public void refreshTransactions() {
                 coins.add(cryptoAccountTypeInfo.getCoinType());
             }
 
-            mKeyringService.getKeyringsInfo(mSharedData.getEnabledKeyrings(), keyringInfos -> {
-                List<AccountInfo> accountInfos =
-                        WalletUtils.getAccountInfosFromKeyrings(keyringInfos);
-                new SelectedAccountResponsesCollector(
-                        mKeyringService, mBraveWalletService, coins, accountInfos)
+            mKeyringService.getAllAccounts(allAccounts -> {
+                new SelectedAccountResponsesCollector(mKeyringService, mBraveWalletService, coins,
+                        Arrays.asList(allAccounts.accounts))
```

Comment:
KeyringService.getAllAccounts will be the only getter for account list and returns all accounts at once(+ some more related info in nearest future)